app/views/devise/passwords/new.html.erb

```diff
@@ -1,15 +1,20 @@
-<h2>Forgot your password?</h2>
+<div class="container">
+  <h2 class="text-center">Forgot your password?</h2>
 
-<%= simple_form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :post }) do |f| %>
-  <%= f.error_notification %>
+  <div class="row">
```

Comment:
raf what does that mean?

Reply:
Hey man! It's a row I put so the size would be better with cols